helm/cluster/templates/clusterapi/controlplane/_helpers_initconfiguration.tpl

```diff
@@ -12,6 +12,7 @@ nodeRegistration:
     {{- end }}
     cloud-provider: external
     healthz-bind-address: 0.0.0.0
+    node-cidr-mask-size: {{ $.Values.global.connectivity.network.pods.nodeCidrMaskSize | default 24 }}
```

Comment:
No need to hard-code as long as nobody is setting the value to null elsewhere, right?

Reply:
right, stupid copilot

Reply:
fixed, I removed it the first time then changed the value from internal to this one and forgot to remove it from the new generated code :D